### ISSUE TITLE
DetailsList focus is not following the active item when NVDA is on

### ISSUE THREAD

**Issue by v-schunduri** [microsoft/fluentui#25182]
### Environment Information

- **Package version(s)**: Fluent UI version 8

### Describe the issue:

We are trying to read the content of Fluent UI's DetailsList using NVDA as a screen reader.

When NVDA is off, all the DetailsList rows and the input controls in them are accessible using arrow keys. Also, the focus follows the current active item. But, when NVDA is on, and navigated to the first input element of the DetailsList first row, the focus stays on the first input element and stops moving with the active item that is being read by the screen reader.

<img width="833" alt="image" src="https://user-images.githubusercontent.com/115549715/195322780-207e0f4d-917a-40c9-9008-7b69c6d77559.png">

In the above image, though the screen reader is on the last column of second row, the focus is still on the first row's Name column which has a link in it. 

#### Actual behavior:
The focus is fixed at the DetailsList first row's first input element when NVDA is on.

#### Expected behavior:
The element focus should follow the item NVDA is currently reading.

Reference codepen link: https://codepen.io/madhurichunduri06/pen/YzLdKYV

**Comment by smhigley** [comment]
It sounds like you're navigating with NVDA's browse mode, which intercepts all keyboard interaction and uses them to control NVDA, not the web browser. When you use arrow keys in NVDA's browse mode, you're moving the NVDA cursor, not page focus.

As a side note, the expected NVDA table commands are ctrl + alt + arrow keys. It would not be common for a screen reader user to move through a table with unmodified arrow keys, though using ctrl + alt + arrow keys will also not move focus (nor is it expected or desired for it to do so).

If you wish to move keyboard focus with NVDA or any other Windows screen reader running, switch your screen reader to forms/application mode.

**Comment by v-schunduri** [comment]
Hi @smhigley, I rechecked the scenario and made sure that I'm not in NVDA's browse mode, the issue still persists. I also observed that except the DetailsList rows, rest all elements in the page are accessed properly and the focus moves along with the active element. I therefore think that I am not in NVDA's browse mode.

With Ctrl+Alt+Arrow keys, putting the focus aside(which is not moving), the Details list rows are still not accessible. 

I assume NVDA does not offer a mode-switch option for the forms/application mode. Nevertheless, users must be able to access the Details List content without having to switch modes.
